common/views/components/Contributor/Contributor.js

```diff
@@ -30,7 +30,9 @@ const Contributor = ({
         <div style={{minWidth: '78px'}} className={spacing({ s: 2 }, { margin: ['right'] })}>
           {contributor.type === 'people' && <Avatar imageProps={imageProps} />}
           {contributor.type !== 'people' &&
-            <Image {...imageProps} extraClasses={'width-inherit'} />
+            <div style={{width: '72px'}}>
```

Comment:
This'll have been the IE11 bug – could you also chuck `maxWidth: '72px'` on there?

Reply:
Happy to look at this once the page is actually working in IE...

Reply:
Seems it worked?
https://github.com/wellcometrust/wellcomecollection.org/pull/3601